app/controllers/application_controller.rb

```diff
@@ -59,11 +60,12 @@ def devise_parameter_sanitizer
   end
 
   private
-  def maybe_enqueue_badge_allocator
+  def maybe_enqueue_badge_allocator_or_bridge_troll_recorder
```

Comment:
Assignment Branch Condition size for maybe_enqueue_badge_allocator_or_bridge_troll_recorder is too high. [18.25/15]